src/collective/cover/upgrades/configure.zcml

```diff
@@ -3,5 +3,6 @@
   <include package=".v11" />
   <include package=".v12" />
   <include package=".v13" />
+  <include package=".v14" />
```

Comment:
@rodfersou this upgrade step is not needed; you already did that on `v13` and that's the one to be used on next release.